chain/network/src/network_protocol/mod.rs

```diff
@@ -90,10 +90,15 @@ impl std::str::FromStr for PeerAddr {
     }
 }
 
```

Comment:
can you add comments about these 2 structs - how do they differ, and do we plan to deprecate AccountData in the future?

Reply:
added docs. No, none of them will be deprecated - I've just separated the content from the metadata for implementation purposes.